gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/integrated/RepeatUnrollStrategy.feature

```diff
@@ -42,3 +42,136 @@ Feature: Step - RepeatUnrollStrategy
       | result |
       | v[lop] |
       | v[ripple] |
+
```

Comment:
i think there should be cases that validate what used to produce differing results to include things like `repeat(...limit(1))` because that would prevent regressions in semantics but also help providers who don't rely on TinkerPop's traversal execution engine to catch failures on upgrade.